app/views/shared/_display_ad.html.erb

```diff
@@ -32,6 +32,15 @@
             <%= I18n.t("display_ad.menu.manage_preferences") %>
           </a>
         </li>
+        <% if Page.exists?(slug: "billboards") %>
+          <hr />
+          <li>
+            <a class="crayons-link crayons-link--block" href="/report-abuse?billboard=<%= request.original_url %>?display_id=<%= display_ad.id %>">
```

Comment:
I think the double `?` here will cause some hard-to-predict cross-browser behaviors. Based on the controller above, which wants this to pass a `billboard`, I am thinking this wants to be more like:

```suggestion
            <a class="crayons-link crayons-link--block" href="/report-abuse?billboard=<%= display_ad.id %>">
```